.travis.yml

```diff
@@ -17,50 +17,47 @@
 # along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
-addons:
-  postgresql: "9.4"
-  apt:
-    packages:
-      - nodejs
-
 notifications:
   email: false
 
+# Not needed at the moment since we use Docker.
+# If this kicks in we can try to use the cache
 sudo: false
```

Comment:
The builds should start failing when they implement this in Docker (probably not too soon) and we'll know to update our build